RDKBACCL-1275: wwan0 IP is not available in LTE (cellular) mode

manigandanj opened the pull request:
Reason for change: RDK-WanManager expects wwan0 interface to be up and running
Test procedure: Found that wwan0 is assigned with IPv4 address
Risks: Low

manigandanj committed:
RDKBACCL-1275: wwan0 IP is not available in LTE (cellular) mode

Reason for change: RDK-WanManager expects wwan0 interface to be up and running
Test procedure: Found that wwan0 is assigned with IPv4 address
Risks: Low

Signed-off-by: Manigandan Gopalakrishnan <Manigandan_Gopalakrishnan@comcast.com>

arunkumar-palanichamy reviewed (commented):
Can you please move this link up before WanManager Tr181 Phy status call?

arunkumar-palanichamy reviewed (changes requested):
v_secure_system("ip link set %s up", wan_ifname);
please move this link up before WanManager PhyStatus update

rdkcmf-jenkins commented:
Job started https://jenkins.code.rdkcentral.com/job/manual-verification/4843/

manigandanj commented:
> v_secure_system("ip link set %s up", wan_ifname); please move this link up before WanManager PhyStatus update

Addressed the comment, Please review.
